topics/introduction.adoc

```diff
@@ -12,12 +12,7 @@ Customer Content Services (CCS) is a Red Hat team responsible for producing docu
 
 In principle, single-streamed documentation will be worked on completely in the upstream and brought down to the Customer Portal wholesale. In some projects, this process eliminates the overhead of maintaining two separate repositories for documentation content. This framework guides you through the process necessary for deciding if a community is a viable single-stream partner, setting up your repositories, choosing the right tools, and more.
 
-[[ccg-evaluating-the-framework]]
-=== Evaluating the Framework
-
-Placeholder text.
-
-[[ccg-checklist]]
-==== Checklist
+[[ccg-contacts]]
+== Contacts
```

Comment:
I guess this is a placeholder for #67?

Reply:
Yes, I moved it from later in the guide to the Introduction.